src/experiment.rs

```diff
@@ -25,10 +27,14 @@ pub struct Experiment {
     add_pulls: Option<f32>,
 }
 
+impl Experiment {
+}
+
 impl Default for Experiment {
     fn default() -> Self {
         Self {
             fabric: Fabric::default(),
+            physics: AIR_GRAVITY.clone(),
```

Comment:
Do you have to clone here?

Reply:
Since it's a const and all

Reply:
Within the experiment, it has to be mutated.